### PR TITLE
Added more classes

### PR DESCRIPTION
**Pull request by AurisTFG**
Added Trackmania2020 server classes (current latest) and Maniaplanet server (beta from 2013)

### CODE REVIEW

File: Tools/ClassIdTxtFromGbxClassInfo/Properties/launchSettings.json [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
       "commandName": "Project",
-      "commandLineArgs": "GbxClassInfo/Vsk5OnlineLauncher.txt GbxClassInfo/Vsk5Online.txt GbxClassInfo/TrackManiaLauncher.txt GbxClassInfo/TmSunriseLauncher.txt GbxClassInfo/TmNationsESWCLauncher.txt GbxClassInfo/TmUnitedLauncher.txt GbxClassInfo/TmUnited.txt GbxClassInfo/TmForeverLauncher.txt GbxClassInfo/TmForeverServer.txt GbxClassInfo/TmForever.txt GbxClassInfo/TurboConfig.txt GbxClassInfo/Turbo.txt GbxClassInfo/ManiaplanetLauncher.txt GbxClassInfo/ManiaplanetServer.txt GbxClassInfo/ManiaplanetLogs.txt GbxClassInfo/Maniaplanet.txt"
+      "commandLineArgs": "GbxClassInfo/Vsk5OnlineLauncher.txt GbxClassInfo/Vsk5Online.txt GbxClassInfo/TrackManiaLauncher.txt GbxClassInfo/TmSunriseLauncher.txt GbxClassInfo/TmNationsESWCLauncher.txt GbxClassInfo/TmUnitedLauncher.txt GbxClassInfo/TmUnited.txt GbxClassInfo/TmForeverLauncher.txt GbxClassInfo/TmForeverServer.txt GbxClassInfo/TmForever.txt GbxClassInfo/TurboConfig.txt GbxClassInfo/Turbo.txt GbxClassInfo/ManiaplanetServer2013-05-22.txt GbxClassInfo/ManiaplanetLauncher.txt GbxClassInfo/ManiaplanetServer.txt GbxClassInfo/ManiaplanetLogs.txt GbxClassInfo/Maniaplanet.txt GbxClassInfo/TrackmaniaServer.txt"
```
Comment (BigBang1112) [inline Tools/ClassIdTxtFromGbxClassInfo/Properties/launchSettings.json:5]: It is better to put the `ManiaplanetServer2013-05-22.txt` before all the Turbo text files, it better follows the timeline, as Turbo was released in 2016.

File: Tools/ClassIdTxtFromGbxClassInfo/Properties/launchSettings.json [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
+      "commandLineArgs": "GbxClassInfo/Vsk5OnlineLauncher.txt GbxClassInfo/Vsk5Online.txt GbxClassInfo/TrackManiaLauncher.txt GbxClassInfo/TmSunriseLauncher.txt GbxClassInfo/TmNationsESWCLauncher.txt GbxClassInfo/TmUnitedLauncher.txt GbxClassInfo/TmUnited.txt GbxClassInfo/TmForeverLauncher.txt GbxClassInfo/TmForeverServer.txt GbxClassInfo/TmForever.txt GbxClassInfo/TurboConfig.txt GbxClassInfo/Turbo.txt GbxClassInfo/ManiaplanetServer2013-05-22.txt GbxClassInfo/ManiaplanetLauncher.txt GbxClassInfo/ManiaplanetServer.txt GbxClassInfo/ManiaplanetLogs.txt GbxClassInfo/Maniaplanet.txt GbxClassInfo/TrackmaniaServer.txt > ../../../../../Resources/ClassId.txt"
```
Comment (BigBang1112) [inline Tools/ClassIdTxtFromGbxClassInfo/Properties/launchSettings.json:9]: Here too

**Comment by AurisTFG** [comment]
Fixed 👍

**Comment by BigBang1112** [comment]
Thanks for the help!

**Pull request merged**